extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -644,6 +644,16 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
             // Ensure that "auto" doesn't appear anywhere here:
             expect(url).to.match(/sz=[0-9]+x[0-9]+%7C1x2%7C3x4&/));
         });
+    it('has correct sz with fluid as multi-size', () => {
+      element.setAttribute('width', '300');
+      element.setAttribute('height', '300');
```

Comment:
This test would be slightly better with different width and height numbers so we could be sure we weren't confusing the two.

Reply:
Done.